nrm_django/grants/admin.py

```diff
@@ -39,131 +39,157 @@ class GrantAdmin(admin.ModelAdmin):
         "created_in_instance",
         "modified_date",
         "status",
+        "wppp_status",
     ]
-    fieldsets = (
+    fieldsets = [
         (
-            "Required",
+            "",
             {
                 "fields": (
-                    ("applicant_name", "proj_cfda_no"),
-                    ("application_type", "app_submission_type"),
-                    ("app_submit_date", "app_received_date"),
+                    "proj_title",
                     ("proposed_start_date", "proposed_end_date"),
-                    ("proj_title", "status"),
-                    "progrm_responsibility_type",
-                )
-            },
-        ),
-        (
-            "Details",
-            {
-                "fields": (
-                    ("gid",),
-                    ("proj_desc", "areas_effected"),
+                    ("applicant_name",),
+                    # Org goes here
+                    ("progrm_responsibility_type",),
+                    # program categories
+                    ("app_submit_date", "app_received_date"),
+                    # Financial assistance
```

Comment:
this one is a simple boolean in @cmajel 's mocks, so I don't think it's another table, but I also can't find a field it maps to. May be a Jamie question.

Reply:
I don't know, I think Jamie needs to answer that.

Reply:
@cmajel can you ask about this when y'all are looking at the Figma mocks today?